worth2/templates/goals/goal_setting_block.html

```diff
@@ -1,14 +1,14 @@
 {% load bootstrap %}
 {% load avatar %}
 
-<div class="goal-setting">
+<div class="goal-setting" id="goal-setting-block">
```

Comment:
there won't be multiple goals on a page correct? ids can get tricky in that case.